conf/nomad/postgres.hcl

```diff
@@ -49,6 +49,14 @@ job "${service_name}" {
       connect {
         sidecar_service {}
       }
+
+      sidecar_task {
+        driver = "docker"
+        resources {
+          cpu    = 500
+          memory = 1000
```

Comment:
Do you have any metrics for this, or is it just "random" values?

Reply:
This is based on examples from the documentation.
It is not tested with a lower value.

Reply:
From what documentation?

Reply:
In Nomad, for sidecar_task.
One example shows cpu = 500 and memory = 1024.  Why 1000 was chosen instead of 1024 is just because the developer who implemented this decided to test with that value.

Reply:
This one?
https://www.nomadproject.io/docs/job-specification/sidecar_task#sidecar_task-examples

Would it be possible to check with the metrics to see if we can lower it?

Reply:
We can extract this as a variable, and delegate setup to the user.

Reply:
> We can extract this as a variable, and delegate setup to the user.

Any idea how much it would differ from system to system? If it would be something that would regularly need tuning I definitely agree.

Reply:
Yes, that's the one.

Reply:
HM's PR includes the resources as variables, you could just use the same implementation: https://github.com/fredrikhgrelland/terraform-nomad-hive/pull/55

Reply:
> HM's PR includes the resources as variables, you could just use the same implementation: [fredrikhgrelland/terraform-nomad-hive#55](https://github.com/fredrikhgrelland/terraform-nomad-hive/pull/55)

I'm doing a test-run on this now.